CONTRIBUTING.md

```diff
@@ -184,13 +184,6 @@ Use the following pattern:
 </record>
 ```
 
-These rules should be followed even when a view is inherited, the module name
-prevents xid conflicts. In the case where a view inherited has a name which
-does not follow the guidelines set above, prefer naming the inherited view
-after the original over using a name which follows the guidelines. This eases
-looking up the original view and other inheritance if they all have the same
-name.
-
```

Comment:
@dreispt look at the end of this section https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#security-view-and-action and at the beginning of the next one